src/PrestaShopBundle/Bridge/Smarty/HeaderHydrator.php

```diff
@@ -0,0 +1,354 @@
+            ]
+        );
+
+        $controllerConfiguration->templatesVars['displayBackOfficeTop'] = $this->getDisplayHookRender(
+            $this->hookDispatcher->dispatchRenderingWithParameters('displayBackOfficeTop', [])
+        );
+        $controllerConfiguration->templatesVars['displayBackOfficeEmployeeMenu'] = $menuLinksCollections;
+        $controllerConfiguration->templatesVars['submit_form_ajax'] = (int) Tools::getValue('submitFormAjax');
+
+        $tabs = $this->getTabs($controllerConfiguration);
+        $currentTabLevel = 0;
+        foreach ($tabs as $tab) {
+            $currentTabLevel = isset($tab['current_level']) ? $tab['current_level'] : $currentTabLevel;
+        }
+
+        $controllerConfiguration->templatesVars['bo_query'] = Tools::safeOutput(Tools::stripslashes(Tools::getValue('bo_query')));
+        $controllerConfiguration->templatesVars['collapse_menu'] = isset($this->cookie->collapse_menu) ? (int) $this->cookie->collapse_menu : 0;
+        $controllerConfiguration->templatesVars['default_tab_link'] = $this->link->getAdminLink(Tab::getClassNameById((int) $controllerConfiguration->user->getData()->default_tab));
+        $controllerConfiguration->templatesVars['employee'] = $controllerConfiguration->user->getData();
+        $controllerConfiguration->templatesVars['help_box'] = Configuration::get('PS_HELPBOX');
+        $controllerConfiguration->templatesVars['is_multishop'] = Shop::isFeatureActive();
+        $controllerConfiguration->templatesVars['login_link'] = $this->link->getAdminLink('AdminLogin');
+        $controllerConfiguration->templatesVars['logout_link'] = $this->link->getAdminLink('AdminLogin', true, [], ['logout' => 1]);
+        $controllerConfiguration->templatesVars['multi_shop'] = Shop::isFeatureActive();
+        $controllerConfiguration->templatesVars['quick_access'] = QuickAccess::getQuickAccessesWithToken($this->language->id, (int) $controllerConfiguration->user->getData()->id);
+        $controllerConfiguration->templatesVars['round_mode'] = Configuration::get('PS_PRICE_ROUND_MODE');
+        $controllerConfiguration->templatesVars['base_url'] = $this->shop->getBaseURL(true);
+        $controllerConfiguration->templatesVars['bootstrap'] = $controllerConfiguration->bootstrap;
+        $controllerConfiguration->templatesVars['controller_name'] = $controllerConfiguration->controllerNameLegacy;
+        $controllerConfiguration->templatesVars['country_iso_code'] = $this->country->iso_code;
+        $controllerConfiguration->templatesVars['currentIndex'] = $this->router->generate('admin_features_index');
+        $controllerConfiguration->templatesVars['current_tab_level'] = $currentTabLevel;
+        $controllerConfiguration->templatesVars['default_language'] = (int) Configuration::get('PS_LANG_DEFAULT');
+        $controllerConfiguration->templatesVars['full_language_code'] = $this->language->language_code;
+        $controllerConfiguration->templatesVars['full_cldr_language_code'] = 'fr';
```

Comment:
```suggestion
        $controllerConfiguration->templatesVars['full_cldr_language_code'] = 'fr';
```

watch out for this kind of stuff